creator-node/src/middlewares.js

```diff
@@ -83,6 +79,12 @@ async function ensurePrimaryMiddleware (req, res, next) {
   } catch (e) {
     return sendResponse(req, res, errorResponseServerError(e))
   }
+
+  // Special case for UserMetadata promotion - unblocks legacy non-creator users writing to UM
+  if (creatorNodeEndpoints.length === 0 && config.get('isUserMetadataNode')) {
```

Comment:
should we make this a falsey check instead of checking length === 0? like, could it be empty string or null instead

Reply:
i think it's better to be explicit here right? when would it be an empty string or null? pretty sure it can never be a string from the function below

Reply:
i don't actually see the benefit of being more explicit here, seemingly we would want to take the same action whether the value is '', null, or [] right?

Reply:
https://codepen.io/arosenb2/post/is-a-javascript-empty-array-falsey-or-not i'd rather check all 3 than make a falsey assumption for this array i think

Reply:
oh wow i forgot that [] is truthy 🤦‍♂️ ok yeah sg